coding/string_utf8_multilang.cpp

```diff
@@ -89,6 +89,12 @@ constexpr bool IsSupportedLangCode(int8_t langCode)
 }
 }  // namespace
 
+bool StringUtf8Multilang::IsServiceLang(std::string_view lang) {
+    return lang == kLanguages[kDefaultCode].m_code
```

Comment:
Please double-check that you follow our style guide. It is described in docs folder.
```suggestion
bool StringUtf8Multilang::IsServiceLang(std::string_view lang)
{
  return lang == kLanguages[kDefaultCode].m_code
```